biome.jsonc

```diff
@@ -80,7 +80,23 @@
         "useComponentExportOnlyModules": "off",
         // We use this operator a lot
         "noNonNullAssertion": "off",
-        "useDefaultSwitchClause": "error"
+        "useDefaultSwitchClause": "error",
+        // Use deep imports to avoid dependence on tree shaking
+        "noRestrictedImports": {
+          "level": "warn",
+          "options": {
+            "paths": {
+              "react-bootstrap": "Please import from 'react-bootstrap/Component' instead to reduce bundle size.",
+              "@fortawesome/free-solid-svg-icons": "Please import from '@fortawesome/free-solid-svg-icons/IconName' instead to reduce bundle size."
+            },
+            "patterns": [
+              {
+                "group": "react-bootstrap/esm/*",
```

Comment:
(FTR, vscode likes to auto-import from these esm/ paths, which is why we ended up with some of them)